zoo/analytics/tasks/gitlab_ci.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+import yaml
+
+from . import GITLAB_CI, OS
+
+
+def parse_gitlab_ci_template(parsed_yaml):
+    """Parse gitlab-ci.yml file for names of templates."""
+    for name in parsed_yaml.get("include", {}):
+        yield GITLAB_CI(Path(name).stem)
+
+
+def get_image(image_def):
+    """Get names of docker images from yaml dict."""
+    if image_def is None:
+        return None
+
+    name, version = image_def.split(":")
```

Comment:
What happens when there is an image without tag? like `image: python`. In this case the default tag for `docker` is `latest`, so we should also return that